### PR TITLE
FOUR-18970: Create the view for show the summary related to the case - History Tab

### PR DESCRIPTION
**Pull request by fagubla**
## Issue & Reproduction Steps
Create the view for show the summary related to the case - History Tab

## Solution
- List the changes you've introduced to solve the issue.

## How to Test
Describe how to test that this solution works.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-18970

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-23b8138f9f.engk8s.processmaker.net

File: resources/jscomposition/cases/casesDetail/edit.js [hunk 1]
```diff
@@ -1,8 +1,366 @@
...
+        Vue.set(obj, prop, value);
+      } else if (descriptor && obj[prop] !== value) {
+        Vue.set(obj, prop, value);
+      }
+    },
+    /**
+     * Listen for Request updates.
+     *
+     */
+    listenRequestUpdates() {
```
Comment (henryjonathanquispe) [inline resources/jscomposition/cases/casesDetail/edit.js:258]: Is necesary this method in the new implementation?

File: resources/jscomposition/cases/casesDetail/edit.js [hunk 2]
```diff
@@ -1,8 +1,366 @@
...
+          this.updateModel(this.request, attribute, response.data[attribute]);
+        }
+        this.refreshTasks++;
+      });
+    },
+    /**
+     * Update a model property.
+     *
+     */
+    updateModel(obj, prop, value, defaultValue) {
```
Comment (henryjonathanquispe) [inline resources/jscomposition/cases/casesDetail/edit.js:244]: Where do we use this method?
 Is necesary this method in the new implementation?

File: resources/jscomposition/cases/casesDetail/edit.js [hunk 3]
```diff
@@ -1,8 +1,366 @@
...
+
+Vue.component("Timeline", Timeline);
 
 new Vue({
   el: "#case-detail",
   components: { CaseDetail },
-  data() {},
+  data() {
+    return {
+      activeTab: "pending",
```
Comment (henryjonathanquispe) [inline resources/jscomposition/cases/casesDetail/edit.js:12]: Where do we use this variables in the template?
 Is necesary this variables in the new implementation?

File: resources/jscomposition/cases/casesDetail/edit.js [hunk 4]
```diff
@@ -1,8 +1,366 @@
...
+      processId: processId,
+      canViewComments: canViewComments,
+      isObjectLoading: false,
+      showTree: false,
+      showTabs: true,
+      showInfo: true,
+    };
+  },
+  computed: {
+    activeErrors() {
```
Comment (henryjonathanquispe) [inline resources/jscomposition/cases/casesDetail/edit.js:43]: Where do we use this computed variables in the template?
 Is necesary this variables in the new implementation?

**Review by henryjonathanquispe — APPROVED** [review]

**Pull request merged**